### DISCUSSION TITLE
Confused about Decrease/Gain of Dependent Continuant

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#357] [Announcements]
Decrease of Generically Dependent Continuant has the comment
>For the most part Generically Dependent Continuants do not inhere in their bearers over some continuous scale. The primary exception is religion and this class allows annotation of those cases where an Agent is described as becoming less religious. Other cases would include the decrease of an organization's bearing of some objective.

- I  can't make sense of the first sentence. 
- In the second sentence, the degree of religiosity of a person is a specifically dependent continuant, a *concretization* of the GDC, not the GDC itself. It is the *individual's* religiosity you are referring to.
- I don't understand the second case, decrease of an organization's bearing of some objective. Could you give an example?

I don't really understand decrease of realizable entity or its subclasses. Could you give examples of instances of each? 

Similar issues with the corresponding increases.

The definition and comment on Gain of Generically Dependent Continuant sounds like you mean there's another concretization of the generic dependent continuant. It might be better titled "Gain of *concretization* of generically dependent continuant". There is no new generically dependent continuant involved.  This would be a subclass of Gain of specifically dependent continuant. It's differentia is that the gain means there's another entity on which the GDC depends.

**Comment by nklsbckmnn** [comment]
> I can't make sense of the first sentence.

I think it just means that GDCs "for the most part" are not continously scaled. You either have a binary scale when a GDC can be carried or not carried or a nominal (or ordinal) scale where you different nominal values (determinates, subtypes) of a GDC (determinable) that can be carried (and that might be ordered).

> In the second sentence, the degree of religiosity of a person is a specifically dependent continuant, a concretization of the GDC, not the GDC itself. It is the individual's religiosity you are referring to.

I don't see why a GDC couldn't be continously scaled itself.

Can GDCs not also be determinable? I guess GDCs would then also have to be non-rigid?

> I don't understand the second case, decrease of an organization's bearing of some objective. Could you give an example?

If you have multiple incompatible objectives that need to be balanced against each other, one of the objectives might become more or less important compared to the others.

>  The definition and comment on Gain of Generically Dependent Continuant sounds like you mean there's another concretization of the generic dependent continuant. 

This interpretation sounds right to me and the implications make sense.

**Comment by michaelrabenberg** [comment]
In light of the 1.6 update, which touched on some Gain/Loss/Increase/Decrease classes, I looked for some information about these classes:

Gain of Dependent Continuant
Loss of Dependent Continuant
Increase of Dependent Continuant
Decrease of Dependent Continuant,

which all refer to a no-longer-recognized BFO class (Dependent Continuant) in their labels and in their definitions (with capital letters). Figured I'd ask about them in this discussion, rather than make a new one with a nearly identical title. 

Is there a reason why these classes and their definitions are still in CCO?
